e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -11,11 +11,12 @@ describe("banner", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
-    cy.visit("/");
-    cy.findByRole("main").findByText("Loading...").should("not.exist");
   });
 
   it("should show a database prompt banner when logged in as an admin, an instance is on a paid plan, and only have a single sample dataset", () => {
+    cy.visit("/");
```

Comment:
Needs to move the `visit()` part out, because I need to call an API before visiting the page in the new test.